.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -270,8 +330,20 @@ jobs:
             cat /usr/local/lib/igc/IGCTAG.txt
           fi
 
+    - name: Download E2E Binaries
+      if: inputs.e2e_binaries_artifact != '' && github.event_name != 'workflow_run'
```

Comment:
Why do we need second part of the expression?

Reply:
I'm not too certain, I was basing this bit off of the `Download SYCL toolchain` step. Looks like that was added in this commit 936dac7965abec530588aef1e73a1b66a9f1a6fd.

Reply:
Ok, i removed it and tried it on the standalone mode here https://github.com/intel/llvm/actions/runs/12361572315/job/34499021175 It didn't try to run this step so i think its fine.